grr-ui/src/templates/AssetMgmt/AssetMgmt.js

```diff
@@ -11,9 +12,8 @@ class AssetMgmt extends Component {
     constructor(props){
         super(props);
         this.toggleVRMode = this.toggleVRMode.bind(this);
-        this.state = {vrMode: false, video:null, width: null, height: null, canvas: null};
-
-
+        this.selectBackground = this.selectBackground.bind(this);
```

Comment:
Is this line needed? I thought the class instance's scope is already bound to all methods and constructors in es6 classes.